### PR TITLE
Add changes to styling and template

### PR DESCRIPTION
**Pull request by OnaMosimege**
Add changes to styling and template according to new spec, and refactor css

### CODE REVIEW

File: app/templates/app/projects.html [hunk 1]
```diff
@@ -3,105 +3,118 @@
...
-                    <option value="">{% trans "Subject" %}</option>
-                    {% for subject in subjects %}
-                      <option value="{{ subject.id }}" {% if request.GET.subject == subject.id|stringformat:"s" %}selected{% endif %}>
-                        {{ subject.name }}
-                      </option>
-                    {% endfor %}
-                </select>
-            </div>
+    <div class="filter-label">
+      <label for="filter">{% trans "Filter:" %}</label>
```
Comment (friedelwolff) [inline app/templates/app/projects.html:10]: There is no element with id="filter".

File: app/templates/app/projects.html [hunk 2]
```diff
@@ -3,105 +3,118 @@
...
-                      <option value="{{ language.id }}" {% if request.GET.language == language.id|stringformat:"s" %}selected{% endif %}>
-                        {{ language.name }}
-                      </option>
-                    {% endfor %}
-                </select>
-            </div>
+    <div class="form-group">
+        <label for="subject_select">{% trans "Subject" %}</label>
+        <select name="subject" class="form-control form-control-lg" id="subject_select">
+            <option value="">{% trans " All subjects" %}</option>
```
Comment (friedelwolff) [inline app/templates/app/projects.html:16]: Please remove unnecessary space at the start of the string.

File: app/templates/app/projects.html [hunk 3]
```diff
@@ -3,105 +3,118 @@
...
-
-      <br />
-      <br />
-      <br />
-      {% endfor %}
     </div>
+  </div>
+  <div class="project-body">
+    <div class="card-text-description">
+        {{ project.description|truncatewords:30 }} <a href="{% url 'project_detail' project.project.id %}">Read more</a>
```
Comment (friedelwolff) [inline app/templates/app/projects.html:89]: Please mark "Read more" for translation.

Comment (OnaMosimege) [reply app/templates/app/projects.html:89]: will do

File: app/templates/app/projects.html (hunk 2 above)
Comment (OnaMosimege) [reply app/templates/app/projects.html:16]: will do

File: app/templates/app/projects.html (hunk 1 above)
Comment (OnaMosimege) [reply app/templates/app/projects.html:10]: will remove

**Review by friedelwolff — APPROVED** [review]

**Pull request merged**